src/formatMarkup.js

```diff
@@ -143,7 +144,12 @@ export const diffableFormatter = function (markup) {
     }
 
     // <tags and="attributes" />
-    let result = '\n' + '  '.repeat(indent) + '<' + node.nodeName;
+    let result = '';
+    if (preChildElementCount == 0) {
+      result = '\n' + ''.repeat(indent) + '<' + node.nodeName;
```

Comment:
```js
'foo_' + 'AB'.repeat(4) + '_bar'
```
becomes
```js
'foo_ABABABAB_bar'
```
* * *
```js
'foo_' + ''.repeat(200) + '_bar'
```
becomes
```js
'foo__bar'
```

Reply:
Interesting. 
`'foo_' + 'AB'.repeat(4) + '_bar'` 
For this, ins't it correct? I thought the answer would come out as "foo_ABABABAB_bar"?

Reply:
I'm trying to point out what `repeat` does, and why you changing it to repeat an empty string, makes no sense.

Reply:
I updated the original comment to better convey this point.